core/dbt/adapters/base/impl.py

```diff
@@ -615,19 +612,21 @@ def get_missing_columns(
         to_relation.
         """
         if not isinstance(from_relation, self.Relation):
-            invalid_type_error(
+            raise InvalidMacroArgType(
                 method_name="get_missing_columns",
                 arg_name="from_relation",
                 got_value=from_relation,
                 expected_type=self.Relation,
+                version="0.13.0",
```

Comment:
Can we get rid of the "version" piece entirely, including in the message? We haven't supported version 0.13.0 in forever, so just a "this isn't supported" should be fine.